site/styles/_buttons.scss

```diff
@@ -230,3 +230,16 @@
     margin-right: 5px;
   }
 }
+
+.btn-block .s72-btn {
+  @extend .btn-block;
+}
+
+.btn-light {
+  background-color: $gray-300;
```

Comment:
I think these grays could have problems on some clients body colors - they probably need to be configurable.

EDIT: I don't know where you are using them so you might have it on some other element/background color - I haven't seen any designs.

Reply:
Bootstrap makes this configurable? e.g. change to $gray-100,  $gray-900 etc per client. Or change .btn-light to use a custom bg color.

This shade of gray works for at least stark white/black.

Reply:
By configurable I meant defining a custom variable for it, e.g. `$button-light-bg-color: $gray-300`. Otherwise the only way to change it is override with custom CSS e.g. `.btn-light { background-color: #ff0033; }`.